### PR TITLE
Fix NanoVDB CI

### PR DESCRIPTION
**Pull request by apradhana**
This is an attempt to fix NanoVDB CI. For Windows, this boils down to downgrading the CMake to `cmake 3.28`. I follow an example given on[ this page ](https://github.com/marketplace/actions/actions-setup-cmake) to downgrade the CMake version. Perhaps I should also update the documentation here: https://www.openvdb.org/documentation/doxygen/dependencies.html#depKnownIssues.

I tried the fix that @Idclip suggested for MacOS, but the CI is still failing when the homebrew installs the dependencies.

### CODE REVIEW

**Review by kmuseth — APPROVED** [review]
looks good to me

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -151,6 +151,12 @@ jobs:
       fail-fast: false
     steps:
     - uses: actions/checkout@v3
+    - name: Setup cmake
+      uses: jwlawson/actions-setup-cmake@v2
+      with:
+        cmake-version: '3.28.x'
```
Comment (Idclip) [inline .github/workflows/build.yml:157]: I'd rather not use random people's actions here

**Comment by Idclip** [comment]
@apradhana thanks for trying to look into this - but are you saying we only support CMake version up to 3.28 with this change? That doesn't sounds appropriate to me, can you link me to any files issues with the newer versions of CMake and zlib through vcpkg please

**Comment by Idclip** [comment]
@apradhana the issue you linked is talking about a regression in CMake 3.27. Your PR seems to be back-porting CMake from 3.29 to 3.28. I'm not sure why your change is related to this.

It also looks like more recent Windows builds have been succeeding so I believe this has been fixed on vcpkg mainline.

Currently all our CI runners are offline (I have no idea why). When they are back up I will investigate this

**Comment by Idclip** [comment]
All of the ASWF CI runners are still offline

**Comment by apradhana** [comment]
Closing this PR. Will add a new PR on-top of [PR-1837](https://github.com/AcademySoftwareFoundation/openvdb/pull/1837).

**Pull request closed without merging**